config/clients/python/template/.github/workflows/main.yaml.handlebars

```diff
@@ -0,0 +1,101 @@
+name: Build, Test and Publish
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  strategy:
+    fail-fast: false
+    matrix:
+      python-version: ["3.9", "3.10", "3.11", "3.12"]
+
+jobs:
+  fossa:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
Keep this file as non-handlebars (yaml) as npo substitutions are being made in it (also because some parts are being detected as handlebars vars)